sim/midas/src/main/scala/midas/core/SimWrapper.scala

```diff
@@ -246,6 +241,7 @@ class SimWrapper(chAnnos: Seq[FAMEChannelConnectionAnnotation],
   val channelPorts = IO(new SimWrapperChannels(bridgeChAnnos, bridgeAnnos, leafTypeMap))
   val hostReset = IO(Input(Bool()))
   val target = Module(new TargetBox(chAnnos, leafTypeMap))
+  target.suggestName(targetInstName)
```

Comment:
Does anything break if this doesn't give it the right name?

Reply:
Yes, but this is no more brittle than it was before.